udata/core/dataset/rdf.py

```diff
@@ -468,6 +468,13 @@ def title_from_rdf(rdf, url):
             return i18n._("Nameless resource")
 
 
+def access_rights_from_rdf(resource: RdfResource):
+    top_el = resource.value(DCT.accessRights)
+    if top_el:
+        # try to find a structured label or fallback on the raw value
+        return rdf_value(top_el, RDFS.label) or rdf_value(resource, DCT.accessRights)
```

Comment:
I don't think it works on graph such as
```
	"dct:accessRights": {
		"@type": "dct:RightsStatement",
		"@label": {
			"@value": "public"
		}
	},
```
as in https://opendata.normandie.fr/api/feed/dcat-ap/2.0.1.json?

Reply:
Quand je convertis le flux en XML (j'ai plus l'habitude de regarder des XML ces temps-ci), ça me donne ça :-/ J'utilise https://www.easyrdf.org/converter depuis JSON-LD vers XML.

```
    <dc:accessRights>
      <dc:RightsStatement>
      </dc:RightsStatement>
    </dc:accessRights>
```